packages/shared/lib/models/Sync.ts

```diff
@@ -256,13 +255,14 @@ export interface DataRecord extends Timestamps {
 export type LastAction = 'ADDED' | 'UPDATED' | 'DELETED' | 'added' | 'updated' | 'deleted';
 
 interface RecordMetadata {
-    first_seen_at: Date;
-    last_modified_at: Date;
+    first_seen_at: string;
+    last_modified_at: string;
     last_action: LastAction;
-    deleted_at: Date | null;
+    deleted_at: string | null;
```

Comment:
those were never date since they are coming from pg as json encoded date.

Reply:
Should update the types in the node-client too to match these: https://github.com/NangoHQ/nango/blob/master/packages/node-client/lib/index.ts#L83

Reply:
fix in 7c906107

Reply:
knex is doing some transformation so I guess it was kinda true :D

Reply:
not even. knex was fine with those fields defined as date but at runtime `typeof` would be `string` 😡